pkg/agent/cniserver/ipam/antrea_ipam.go

```diff
@@ -48,13 +48,19 @@ var antreaIPAMDriver *AntreaIPAM
 // Resource needs to be unique since it is used as identifier in Del.
 // Therefore Container ID is used, while Pod/Namespace are shown for visibility.
 // TODO: Consider multi-interface case
-func getAllocationOwner(args *invoke.Args, k8sArgs *argtypes.K8sArgs) crdv1a2.IPAddressOwner {
+func getAllocationOwner(args *invoke.Args, k8sArgs *argtypes.K8sArgs, preallocateOwner *crdv1a2.IPAddressOwner) crdv1a2.IPAddressOwner {
+	podOwner := &crdv1a2.PodOwner{
+		Name:        string(k8sArgs.K8S_POD_NAME),
+		Namespace:   string(k8sArgs.K8S_POD_NAMESPACE),
+		ContainerID: args.ContainerID,
+	}
+	if preallocateOwner != nil {
```

Comment:
perhaps `reserve` rather than `preallocate` is a more accurate term here? Since with initial support IPs are not preallocated.

Reply:
I think we can keep this name, since we'll support preallocate in future and the `phase` is set to `preallocated` when StatefulSet IP released.
Do you think we should change `IPAddressPhasePreallocated` to `IPAddressPhaseReserved` in CRD def and refactor all the reference?

Reply:
Yes, since `preallocated` term may be misleading

Reply:
I also feel `reserved `sounds better.

Reply:
Move to "reserved".